modules/ROOT/pages/anypoint-platform-cli-commands.adoc

```diff
@@ -1380,18 +1459,23 @@ This command does not take any options, except for the default ones: `--help`, `
 
 This command displays information on the application you pass in `<name>`. +
 You can start typing your application's name and press `tab` for Anypoint Platform CLI to autocomplete it, or you can double tap `tab` for a full list of all the values you can pass. +
```

Comment:
Is "double tab" hyphenated? "double-tap"?

Reply:
Yes, but I'm leaving rewriting until next time.